sections/futures/MobileTrade/UserTabs/UserTabs.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import TabButton from 'components/Button/TabButton';
+
+import OpenPositionTab from './OpenPositionTab';
+import OrdersTab from './OrdersTab';
+import TradesTab from './TradesTab';
+import TransfersTab from './TransfersTab';
+
+const TABS = [
+	{
+		title: 'Open Position',
```

Comment:
I think this should be called "Position" since they could have one open already

Reply:
@asherism What do you think?

Reply:
so "Position" and "Open Orders" instead of "Open Position" and "Orders"